Patterns/Dutch/Dutch-DateTime.yaml

```
@@ -1257,6 +1275,7 @@ AmbiguityFiltersDict: !dictionary
   entries:
     '^\d{4}$': '(\d\.\d{4}|\d{4}\.\d)'
     '\b(lunch)$': '(?<!\b(op|om|voor|ana(ar)?|rond)\b\s)(lunch)'
```

```suggestion
    '\b(lunch)$': '(?<!\b(op|om|voor|na(ar)?|rond)\b\s)(lunch)'
```

Not sure what `ana(ar)?` should be, I assume it is 'na' (after).

---

fixed as suggested